edxval/api.py

```diff
@@ -277,17 +278,59 @@ def get_videos_for_ids(
     return (VideoSerializer(video).data for video in videos)
 
 
-def get_video_info_for_course_and_profile(course_id, profile_name):
+def get_video_info_for_course_and_profiles(course_id, profiles):
     """
-    Returns a dict mapping profiles to URLs.
+    Returns a dict of edx_video_ids with a dict of requested profiles.
 
     If the profiles or video is not found, urls will be blank.
```

Comment:
This comment doesn't make sense; if a profile is missing, it will simply not be present. We should also note that a video with no encodings will not be included in the result.